aggregator/types.go

```diff
@@ -129,6 +129,24 @@ type Handler interface {
 type Options interface {
 	/// Read-write base options
 
+	// SetDefaultCounterAggregationTypes sets the counter aggregation types.
+	SetDefaultCounterAggregationTypes(aggTypes policy.AggregationTypes) Options
```

Comment:
nit: the convention is simply name the parameter `value` regardless of what the actual type is.